vast/vastutil/unwrap.go

```diff
@@ -63,13 +65,12 @@ func unwrap(ctx context.Context, v *vast.Vast, unwrappedList []*vast.Vast, ua, i
 	}
 	defer func() {
 		if resp != nil && resp.Body != nil {
-			resp.Body.Close()
+			_, _ = io.Copy(ioutil.Discard, resp.Body)
+			_ = resp.Body.Close()
 		}
 	}()
 
-	if err != nil {
-		return nil, err
-	} else if err = xml.NewDecoder(resp.Body).Decode(&innerVast); err != nil {
+	if err = xml.NewDecoder(resp.Body).Decode(&innerVast); err != nil {
```

Comment:
if change to this way would cause the original err was not returned?

Reply:
as it is duplicate with line 63